src/styles/blocks/snake.scss

```
@@ -0,0 +1,48 @@
+.snake {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($sizeBox, 1fr));
+  
+  @media (min-width: 600px) {
+    & {
+      grid-template-columns: repeat(2, 1fr);
+    }
+  }
+
+  @media (min-width: 900px) {
+    & {
+      grid-template-columns: repeat(3, 1fr);
+    }
```

you can remove `&` form selectors
```suggestion
    grid-template-columns: repeat(2, 1fr);
  }

  @media (min-width: 900px) {
    grid-template-columns: repeat(3, 1fr);
```